plugins/Name_ShouldBeHousenumber.py

```diff
@@ -0,0 +1,81 @@
+#-*- coding: utf-8 -*-
+
+###########################################################################
+##                                                                       ##
+## Copyrights arrival-spring 2022                                        ##
+##                                                                       ##
+## This program is free software: you can redistribute it and/or modify  ##
+## it under the terms of the GNU General Public License as published by  ##
+## the Free Software Foundation, either version 3 of the License, or     ##
+## (at your option) any later version.                                   ##
+##                                                                       ##
+## This program is distributed in the hope that it will be useful,       ##
+## but WITHOUT ANY WARRANTY; without even the implied warranty of        ##
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         ##
+## GNU General Public License for more details.                          ##
+##                                                                       ##
+## You should have received a copy of the GNU General Public License     ##
+## along with this program.  If not, see <http://www.gnu.org/licenses/>. ##
+##                                                                       ##
+###########################################################################
+
+from modules.OsmoseTranslation import T_
+from plugins.Plugin import Plugin
+import regex as re
+
+class Name_ShouldBeHousenumber(Plugin):
+
+    def init(self, logger):
+        Plugin.init(self, logger)
+        self.errors[804] = self.def_class(item = 5010, level = 1, tags = ['name', 'fix:chair'],
```

Comment:
Just a minor remark: is level 1 (high priority) appropriate? I would personally go for level 2 or 3

Reply:
I'm not sure I know enough about levels to have an opinion

Reply:
Yes, not level 1. 2 look good for me.

Reply:
Done